README.md

```diff
@@ -0,0 +1,148 @@
+[tests_link]: https://github.com/dimagi/django-field-audit/actions/workflows/tests.yml
+[coverage_badge]: https://github.com/dimagi/django-field-audit/raw/coverage-badge/coverage.svg
+[coverage_link]: https://github.com/dimagi/django-field-audit/actions/workflows/coverage.yml
+[pypi_badge]: https://badge.fury.io/py/django-field-audit.svg
+[pypi_link]: https://pypi.org/project/django-field-audit/
+
+A Django app for auditing field changes on database models.
+
+## Installation
+```
+pip install django-field-audit
+```
+
+## Documentation
+
+<!--
+The [django-field-audit documentation][docs] shows how to use this library to
+audit field changes on Django Models.
+
+[docs]: https://dimagi.github.io/django-field-audit/
+-->
+
+### Django Settings
+
+Add the app to your Django `INSTALLED_APPS` configuration to enable the app and
+run migrations. Installed apps settings example:
+
+```python
+INSTALLED_APPS = [
+    # ...
+    "field_audit",
+]
+```
+
+If `changed_by` auditing is desired (enabled by default), add the app middleware
+to your Django `MIDDLEWARE` configuration. For example:
+
+```python
+MIDDLEWARE = [
+    # ...
+    "field_audit.middleware.FieldAuditMiddleware",
+]
+```
+
+#### Custom settings
+
+| Name                              | Description                                                | Default value when unset
+|:----------------------------------|:-----------------------------------------------------------|:------------------------
+| `FIELD_AUDIT_AUDITEVENT_MANAGER`  | A custom manager to use for the `AuditEvent` Model.        | `field_audit.models.DefaultAuditEventManager`
+| `FIELD_AUDIT_FIELDCHANGE_MANAGER` | A custom manager to use for the `FieldChange` Model.       | `django.db.models.Manager`
+| `FIELD_AUDIT_AUDITORS`            | A custom list of auditors for acquiring `changed_by` info. | `["field_audit.auditors.RequestAuditor", "field_audit.auditors.SystemUserAuditor"]`
+
+### Model Auditing
+
+To begin auditing Django models, import the `field_audit.audit_fields` decorator
+and decorate models specifying which fields should be audited for changes.
+Example code:
+
+```python
+# flight/models.py
+
+from django.db import models
+from field_audit import audit_fields
+
+
+@audit_fields("tail_number", "make_model", "operated_by")
+class Aircraft(models.Model):
+    id = AutoField(primary_key=True)
+    tail_number = models.CharField(max_length=32, unique=True)
+    make_model = models.CharField(max_length=64)
+    operated_by = models.CharField(max_length=64)
+```
+
+
+## Contributing
+
+All feature and bug contributions are expected to be covered by tests.
+
+### Setup for developers
+
+Create/activate a python virtualenv and install the required dependencies.
+
+```shell
+cd django-field-audit
+mkvirtualenv django-field-audit  # or however you choose to setup your environment
+pip install django nose flake8 coverage
+```
+
+### Running tests
+
+- Tests
+  ```shell
+  nosetests
```

Comment:
Curious why we are using nose which is no longer maintained. Copying this note from the docs:

> Nose has been in maintenance mode for the past several years and will likely cease without a new person/team to take over maintainership. New projects should consider using [Nose2](https://github.com/nose-devs/nose2), [py.test](http://pytest.org/), or just plain unittest/unittest2.

I would recommend pytest over nose2 (I believe nose2 already recommends using pytest)

Reply:
I see that's already a todo